engine/src/lin_time.cpp

```diff
@@ -88,20 +90,6 @@ void reset_time_compression(const KBData &, KBSTATE a) {
     }
 }
 
-void pause_key(const KBData &s, KBSTATE a) {
-    static bool paused = false;
-    if (a == PRESS) {
-        if (paused == false) {
-            timecompression = .0000001;
-            timecount = 0;
-            paused = true;
-        } else {
-            paused = false;
-            reset_time_compression(s, a);
-        }
-    }
-}
```

Comment:
Q: is this disabling the games ability to Pause? Or is that being handled somewhere else now?

Reply:
This function is unused (AFAIK, I git grep'ed it in engine & assets), I think what is used is the `toggle_pause()` function, a few lines below, in `engine/src/cmd/ai/firekeyboard.cpp`.

Reply:
@vincele thanks - I'll try to pull it down and test to validate. That's my main concern here.